### DISCUSSION TITLE
续费一年变成了免费版

### DISCUSSION THREAD

**Discussion by dlyxy** [getlantern/lantern-forum-cn#790] [General]
续费了一年，今天用发现变成了免费版

**Comment by SLS321** [comment]
我续了两年，今天也变成免费版了，无法登录账号（收不到邮箱登录码）~

**Comment by MaxMKMax** [comment]
@dlyxy @SLS321 您好. 请你们提供我支付单据和邮箱. 谢谢您! 然后我会帮你们提交问题上去. 但我需要信息才能帮你们提交.
